scipy/special/cephes/ellpj.c

```diff
@@ -93,17 +100,44 @@ int ellpj(double u, double m, double *sn, double *cn, double *dn, double *ph)
         *dn = 1.0 - 0.5 * m * t * t;
         return (0);
     }
+
+    /* if m is near 1, use approximation for amplitude
+       given in Abramowitz and Stegun, 16.15.4 */
```

Comment:
Relevant exerpts:

Definition of `m1`.

<img width="400" alt="image" src="https://github.com/user-attachments/assets/c4775755-9aa9-4c53-828d-f7f736c153a9">

`phi` goes by the symbol $am$, and fefinitions of `sn`, `cn`, and `dn` in terms of `phi`:

<img width="400" alt="image" src="https://github.com/user-attachments/assets/c6b307c1-944d-46ff-8d2c-38745ef7795a">

Definition of $\text{gd}$:

<img width="886" alt="image" src="https://github.com/user-attachments/assets/e4ae84a3-ecd9-448d-8249-6bbddc690ced">

16.15.4 itself:

<img width="400" alt="image" src="https://github.com/user-attachments/assets/1eefaf9b-b228-495c-a51f-0a6d83e41c19">

(and note that $\text{sech} = \frac{1}{\cosh}$, leading to the simplification)